src/main/java/edu/greenblitz/tobyDetermined/subsystems/DriveTrain/IO/GyroIOs/GyroIO.java

```diff
@@ -0,0 +1,25 @@
+package edu.greenblitz.tobyDetermined.subsystems.DriveTrain.IO.GyroIOs;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface GyroIO {
+
+    default void setYaw(double yaw) {
```

Comment:
Why define this method as a 'default' if the implementation is empty?

Reply:
you need to define this method as default because if you wanted to replay matches you need to create an empty interface.